aggregator_api/src/tests.rs

```diff
@@ -78,7 +78,7 @@ async fn get_config() {
             r#""protocol":"DAP-09","dap_url":"https://dap.url/","role":"Either","vdafs":"#,
             r#"["Prio3Count","Prio3Sum","Prio3Histogram","Prio3SumVec"],"#,
             r#""query_types":["TimeInterval","FixedSize"],"#,
-            r#""features":["TokenHash","UploadMetrics","TimeBucketedFixedSize","PureDpDiscreteLaplace"],"#,
+            r#""features":["TokenHash","UploadMetrics","TimeBucketedFixedSize","PureDpDiscreteLaplace","AggregationJobMetrics"],"#,
```

Comment:
divviup-api uses the `AggregationJobMetrics` feature to govern whether it expects aggregators to expose these counters. In order to avoid needing features like `AggregationJobMetrics-2` or `-3`, I think that while I may not wire up the probes for all aggregation job counters in this PR, it would be wise to add every field we expect to report to `TaskAggregationCounter` and into the database table. That also means fewer DB migrations.